plugins/legacy-plugin-chart-markup/src/controlPanel.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { t } from '@superset-ui/translation';
+import { validateNonEmpty } from '@superset-ui/validator';
+import { formatSelectOptions } from '@superset-ui/control-utils';
+
+console.log('what are going on with these controls')
+export default {
+  controlPanelSections: [
+    {
+      label: t('Code'),
+      expanded: true,
+      controlSetRows: [
+        [
+          {
+            name: 'markup_type',
+            config: {
+              type: 'SelectControl',
+              label: t('Markup Type'),
+              clearable: false,
+              choices: formatSelectOptions(['markdown', 'html']),
+              default: 'markdown',
+              validators: [validateNonEmpty],
+              description: t('Pick your favorite markup language'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'code',
+            config: {
+              type: 'TextAreaControl',
+              label: t('Code'),
+              description: t('Put your code here'),
+              mapStateToProps: state => ({
+                language:
+                  state.controls && state.controls.markup_type
+                    ? state.controls.markup_type.value
+                    : 'markdown',
+              }),
+              default: '',
+            },
+          },
+        ],
+      ],
+    },
+  ],
+  sectionOverrides: {
```

Comment:
Is this section needed?

Reply:
@kristw I think they are needed (or at least relevant). There are a few plugins that do this... iframe, markup, and separator. It appears this basically removes the `time_range` control  that's seemingly injected automatically in some cases as part of the `druidTimeSeries`. Maybe we can find a better way of addressing this in the future, but it seems out of the scope of this PR, which is just to migrate the code that's there already.

Reply:
got it